olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml

```diff
@@ -274,6 +316,41 @@ spec:
                 - containerPort: 9090
                   name: metrics
               serviceAccountName: knative-serving-operator
+      - name: knative-eventing-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: knative-eventing-operator
+          template:
+            metadata:
+              annotations:
+                sidecar.istio.io/inject: "false"
+              labels:
+                name: knative-eventing-operator
+            spec:
+              containers:
+              - env:
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: knative-eventing-operator
+                - name: SYSTEM_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: METRICS_DOMAIN
+                  value: knative.dev/eventing-operator
+                # TODO
+                image: gcr.io/knative-releases/knative.dev/eventing-operator/cmd/manager@sha256:294a0fcfa6c206f49c56b299c4377d27a8ac420aafa18243874ef8b7e28859c0
+                imagePullPolicy: IfNotPresent
+                name: knative-eventing-operator
+                ports:
+                - containerPort: 9090
+                  name: metrics
+              serviceAccountName: knative-serving-operator
```

Comment:
This serviceAccount is used by the upstream serving operator deployment and our operator deployment too.

I didn't want to create a separate SA just for the upstream eventing operator as we will need our Knative operator deployment to have permissions for both serving and eventing. ...and, we cannot use 2 service accounts in a deployment.

So, I just added eventing related permissions to this service account. It needs to be renamed though (later)

Reply:
Agree, there's still a lot of renaming to be done. We can do that in a later PR.